source/game/scene/RaceScene.cc

```diff
@@ -1,6 +1,7 @@
 #include "RaceScene.hh"
 
 #include "game/field/CollisionDirector.hh"
+#include "game/field/ObjectDirector.hh"
```

Comment:
Bit awkward to address here, but we have started going with the convention that other directories must use angled brackets. I'm fine if we don't address it in this PR (KartMove.cc also violates this)

Reply:
This is a non-issue, as the convention applies to external libraries such as `egg` and `abstract` in this case, not `game`.